src/lib/y2storage/callbacks/libstorage_callback.rb

```diff
@@ -67,7 +67,7 @@ def error(message, what)
         desc = error_description(what)
         hint = _("Click below to see more details (English only).")
         question = _("Continue despite the error?")
-        msg = "#{message}\n\n#{desc}\n\n#{hint}\n\n#{question}"
+        msg = "#{message}\n\n#{desc}\n#{hint}\n\n#{question}"
```

Comment:
Why this change? I would keep it as it was, and simply add `desc = error_description(what).chop` in case we want to ignore the possible `\n`. Otherwise, we have to remember to add a `\n` in all possible results of `#error_description`.